examples/express/utils.js

```diff
@@ -20,6 +20,11 @@ const populateEnv = () => {
   if (!process.env.OTEL_EXPORTER_JAEGER_ENDPOINT) {
     process.env.OTEL_EXPORTER_JAEGER_ENDPOINT = process.env.SIGNALFX_ENDPOINT_URL;
   }
+  if (!process.env.OTEL_EXPORTER_OTLP_ENDPOINT) {
+    process.env.OTEL_EXPORTER_OTLP_ENDPOINT = (
+      process.env.SIGNALFX_ENDPOINT_URL || process.env.OTEL_EXPORTER_JAEGER_ENDPOINT
```

Comment:
I don't think we support SIGNALFX_ENDPOINT_URL in new Otel based libs anymore.

Reply:
I think we should remove these as the libs don't support these fallbacks and it might be confusing to keep them in examples.

Reply:
This whole function is specifically for people migrating as the first preliminary step to be able to..
1. get something that looks like their old config up and running with OTEL;
2. see what needs to be converted and how.

Reply:
I don't think we ever had a goal to have config backward compatibility in Otel for old SFX libs. IMO this can lead to bugs as things will work with SIGNALFX config in this config but may not in another real project. IMO we should restrict the migration guide strictly to markdown documentation and not pollute otel examples with old sfx config. I'm sure this will lead to confusion. 

That said, I see you're just updating existing examples and didn't add new one so it's not a blocker for me for this PR. I still think we should remove references to SFX stuff later and keep pure otel examples here.

Reply:
> I don't think we ever had a goal to have config backward compatibility in Otel for old SFX libs.

Not backwards compatibility, but helping through testing and conversion. I'd rather get some feedback from the first users, but totally open to removing this if it ever causes any confusion.

Reply:
I think we should have separate migration example and keep this one as pure Otel based but dont need it in this PR.